src/utils/web3.js

```diff
@@ -63,9 +63,13 @@ export const web3Connect = () => async (dispatch, getState) => {
 
 export const getSmartContract = (web3, address) => {
   if (!web3) { return null; }
-
-  const abi = require(`../smart_contracts/${address}.json`);
-  return web3.eth.contract(abi).at(address);
+  try {
```

Comment:
Plase use here function `isCorrectTokenAddress ` instead of try/catch construction